docs/TESTS.md

```diff
@@ -0,0 +1,70 @@
+# Test Categories
+
+This is not meant to be an exhaustive list, but it should hopefully shed some
+light on how we categorize tests.
+
+## `/hardening`
+
+These specifically
+
+1. harden the OS installation via some means (`oscap`, Ansible, etc.)
```

Comment:
It look weird when oscap is as code (`oscap`) and Ansible not. I'd prefer it unified